pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes 
+excerpt: Discover the new features, updates, and known limitations in this release of the Confluent Kafka Service
+title: Release Notes
+menuWeight: 10
+model: /services/confluent-kafka/data.yml
+render: mustache
+---
+# Release Notes for Confluent Kafka Service version 2.5.0-4.1.2
+
+## Updates
+
+- Update to Confluent Kafka version 4.1.2
+- Update dcos-coommons sdk version to 0.55.2
```

Comment:
Can you also add a link that points to the upstream changelog? For the user's reference:
https://docs.confluent.io/4.1.2/release-notes.html

Reply:
Done